Dockerfile.iso

```diff
@@ -1,4 +1,4 @@
-ARG OS_IMAGE=registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-node-image/5.2:latest
+ARG OS_IMAGE=registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-node-image/5.3:latest
```

Comment:
Looks like we have been using the wrong image for a while 🤦🏽

Reply:
Yes, that was my comment on a previous PR where you were modifying this file ;-)